README.md

```diff
@@ -314,6 +315,7 @@ Options:
   -enable-url-source        Enable remote HTTP URL image source processing (?url=http://..)
   -enable-placeholder       Enable image response placeholder to be used in case of error [default: false]
   -enable-auth-forwarding   Forwards X-Forward-Authorization or Authorization header to the image source server. -enable-url-source flag must be defined. Tip: secure your server from public access to prevent attack vectors
+  -custom-headers			      Forwards custom headers to the image source server. -enable-url-source flag must be defined.
```

Comment:
The spacing is a bit off here compared to the rest. Even though the Code uses tabs for indentation, the Documentation uses spaces to keep it nicely consistent.